packages/client/components/GitHubFieldMenu.tsx

```diff
@@ -49,37 +108,76 @@ const GitHubFieldMenu = (props: Props) => {
     SprintPokerDefaults.SERVICE_FIELD_NULL
   ] as string[]
   const defaultActiveIdx = defaults.indexOf(serviceFieldName) + 1
+  const {modalPortal, openPortal, closePortal: closeModal} = useModal({
+    id: 'editGitHubLabel',
+    parentId: 'githubFieldMenu'
+  })
 
   if (task?.integration?.__typename !== '_xGitHubIssue') return null
   const {integration} = task
   const {repository} = integration
   const {nameWithOwner} = repository
   const handleClick = (labelTemplate: string) => () => {
-    UpdateGitHubDimensionFieldMutation(atmosphere, {
-      dimensionName,
-      labelTemplate,
-      nameWithOwner,
-      meetingId
-    })
+    if (labelTemplate !== serviceFieldName) {
+      UpdateGitHubDimensionFieldMutation(atmosphere, {
+        dimensionName,
+        labelTemplate,
+        nameWithOwner,
+        meetingId
+      })
+    }
     closePortal()
   }
+  const openEditModal = (e: React.MouseEvent) => {
```

Comment:
Curious whether you prefer to be explicit with React event types vs destructuring them when importing, as we do with `useState`, or whether you don't mind either way?